README.md

```diff
@@ -43,6 +43,12 @@ Run the `run.py` script to perform training / classification. The first time `ru
 
 1. Clone https://hg.mozilla.org/mozilla-central/.
 2. Run `./mach vcs-setup` in the directory where you have cloned mozilla-central.
+3. In order to run the repository mining script, you'll have to enable a few extensions. Add the following to your .hgrc file, under the extensions section:
```

Comment:
`In order to run the repository mining script` is redundant with the title of the section, you can remove it. Just say `Enable the pushlog, hgmo and mozext extensions. For example, if you are on Linux, add the following ...`

Reply:
Replace `.hgrc` with `~/.hgrc`.